test/school_test.rb

```diff
@@ -45,11 +45,34 @@ def test_it_can_add_student_names
   end
 
   def test_it_can_calculate_end_time
-    skip
+
     school1 = School.new('9:00', 7)
     school2 = School.new('9:00', 3)
 
     assert_equal '16:00', school1.end_time
     assert_equal '12:00', school2.end_time
   end
+
+  def test_it_can_check_full_time
+    school = School.new('9:00', 7)
+    school1 = School.new("7:00",3)
+    assert_equal true, school.is_full_time?
+    assert_equal false, school1.is_full_time? 
+
+  end
+
+  def test_standard_student_names_capatilizes
+    school = School.new('9:00', 7)
+    school.add_student_name('Aurora')
+    school.add_student_name('tim')
+    school.add_student_name('megan')
+    assert_equal ["Aurora", "Tim", "Megan"], school.standard_student_names
+  end
+
+  def test_convert_end_time_to_clock_time
```

Comment:
Nitpick, as a rule of thumb you only want one assertion per test. Either line 75 or 76 is good enough here to give me confidence your `#convert_end_time_to_clock_time` works correctly